### ISSUE TITLE
Add option in keymap to exit  ChatGPT window while in normal mode

### ISSUE THREAD

**Issue by Adoliin** [jackMort/ChatGPT.nvim#108]
As of now we have this:
```lua
{
  keymaps = {
    close = { '<C-c>' }
  }
}
```
Which only allow us to map an exit key while in insert mode what i want is something like this:

```lua
{
  keymaps = {
    close = { '<C-c>' }
    close_normal = '<esc>'
  }
}
```

**Comment by cmpadden** [comment]
The `keymaps.close` option is being set for both insert and normal mode, see:

https://github.com/jackMort/ChatGPT.nvim/blob/main/lua/chatgpt/module.lua#L189-L191

```
for _, keymap in ipairs(close_keymaps) do
  chat_input:map("i", keymap, function()
    chat_input.input_props.on_close()
  end, { noremap = true, silent = true })

  chat_input:map("n", keymap, function()
    chat_input.input_props.on_close()
  end, { noremap = true, silent = true })
end
```

I've confirmed that having the following allows me to close the window with `<C-c>` in both modes .

```
close = { "q", "<C-c>" },
```

**Comment by textzenith** [comment]
I think I'll speak for a lot of users when I say that I'd like the prompt to work just like Telescope (i.e. Esc closes entire popup in normal mode)

**Comment by cmpadden** [comment]
@Adoliin & @textzenith this PR https://github.com/jackMort/ChatGPT.nvim/pull/139 adds the `close_normal` option. Let me know if this is what you were expecting. Cheers!

**Comment by Adoliin** [comment]
Yup exactly that's what I was looking for, hopefully it gets merged. Thanks for your contribution.

**Comment by textzenith** [comment]
Legend!

Yes @cmpadden, that's exactly how it works. It's a bit unintuitive at first but it's a UX most of us are familiar with (due to Telescope's popularity) and feels like on of the Vimmiest ways to do things.

edit: I've been thinking some more about this:

In Telescope, you can press Ctrl-C from insert mode, still. But in normal mode it doesn't work — it wants to exit Neovim. Doesn't feel right. I like that in ChatGPT.nvim you can.
In command mode, you can escape just by deleting (hitting backspace) at the start of the prompt. Both Telescope and ChatGPT.nvim don't allow this. Not sure if this feels right.

**Comment by ariel-frischer** [comment]
I would really like mappings for normal mode, and mappings for insert mode separately.
